<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="Suite A" tests="4" failures="2" errors="0" time="2.0">
    <testcase name="test1" classname="SuiteA.TestClass1" time="0.5"/>
    <testcase name="test2" classname="SuiteA.TestClass2" time="0.5">
      <failure message="Test failed" type="AssertionError">First failure in Suite A</failure>
    </testcase>
    <testcase name="test3" classname="SuiteA.TestClass3" time="0.5">
      <failure message="Another failure" type="AssertionError">Second failure in Suite A</failure>
    </testcase>
    <testcase name="test4" classname="SuiteA.TestClass4" time="0.5"/>
  </testsuite>
  <testsuite name="Suite B" tests="3" failures="1" errors="0" time="1.5">
    <testcase name="test1" classname="SuiteB.TestClass1" time="0.5"/>
    <testcase name="test2" classname="SuiteB.TestClass2" time="0.5">
      <failure message="Failed in Suite B" type="AssertionError">Failure in Suite B</failure>
    </testcase>
    <testcase name="test3" classname="SuiteB.TestClass3" time="0.5"/>
  </testsuite>
  <testsuite name="Suite C" tests="2" failures="1" errors="0" time="1.0">
    <testcase name="test1" classname="SuiteC.TestClass1" time="0.5">
      <failure message="Failed in Suite C" type="AssertionError">Failure in Suite C</failure>
    </testcase>
    <testcase name="test2" classname="SuiteC.TestClass2" time="0.5"/>
  </testsuite>
</testsuites>